python/cugraph/cugraph/dask/comms/comms.py

```diff
@@ -265,3 +265,16 @@ def get_n_workers(sID=None, dask_worker=None):
             dask_worker = get_worker()
         sessionstate = get_raft_comm_state(sID, dask_worker)
         return sessionstate["nworkers"]
+
+
+def rank_to_worker(client):
+    """
+    Return a mapping of dask workers to ranks.
```

Comment:
minor: isn't it a mapping of ranks to dask workers (and not the other way around)?

Reply:
Right.

Reply:
Ah, right. I think I had placeholders for "key" and "value" names but I got mixed up. I'll make those corrections